index.md

```diff
@@ -14,7 +14,7 @@ enddate: FIXME        # machine-readable end date for the workshop in YYYY-MM-DD
 instructor: ["FIXME"] # boxed, comma-separated list of instructors' names as strings, like ["Kay McNulty", "Betty Jennings", "Betty Snyder"]
 helper: ["FIXME"]     # boxed, comma-separated list of helpers' names, like ["Marlyn Wescoff", "Fran Bilas", "Ruth Lichterman"]
 email: ["fixme@example.org"]    # boxed, comma-separated list of contact email addresses for the host, lead instructor, or whoever else is handling questions, like ["marlyn.wescoff@example.org", "fran.bilas@example.org", "ruth.lichterman@example.org"]
-collaborative_notes:             # optional: URL for the workshop collaborative notes, e.g. an Etherpad or Google Docs document
```

Comment:
The variable name in the template is `collaborative_notes` (https://github.com/carpentries/workshop-template/blob/cbc5e0c7ca2d5ad935d05418bdfcb8aad6a726bc/index.md#L250-L253). Could you expand on what works/doesn't work with the current variable name? and it might be that we need to update the docs to reflect the accurate variable name.

I like the suggestion of adding an example URL in the comment!